packages/app/src/domain/vaccine/vaccine-coverage-choropleth.tsx

```diff
@@ -200,41 +143,29 @@ export const VaccineCoverageChoropleth = ({
   );
 };
 
-type VaccineCoverageData =
-  | GmCollectionVaccineCoveragePerAgeGroup
-  | VrCollectionVaccineCoveragePerAgeGroup;
+type VaccineCoverageData = GmCollectionVaccineCoveragePerAgeGroup | VrCollectionVaccineCoveragePerAgeGroup;
 
 type ChoroplethTooltipProps<T extends VaccineCoverageData> = {
   data: TooltipData<T>;
   selectedCoverageKind: CoverageKindProperty;
   ageGroups: AgeGroup[];
-  mapData:
-    | GmCollectionVaccineCoveragePerAgeGroup[]
-    | VrCollectionVaccineCoveragePerAgeGroup[];
+  mapData: GmCollectionVaccineCoveragePerAgeGroup[] | VrCollectionVaccineCoveragePerAgeGroup[];
 };
 
-export function ChoroplethTooltip<T extends VaccineCoverageData>({
-  data,
-  ageGroups,
-  selectedCoverageKind,
-  mapData,
-}: ChoroplethTooltipProps<T>) {
+export function ChoroplethTooltip<T extends VaccineCoverageData>({ data, ageGroups, selectedCoverageKind, mapData }: ChoroplethTooltipProps<T>) {
   const { commonTexts } = useIntl();
   const coverageKindsText = commonTexts.vaccinations.coverage_kinds;
-  const ageGroupsText: SiteText['common']['common']['age_groups'] =
-    commonTexts.common.age_groups;
+  const ageGroupsText: SiteText['common']['common']['age_groups'] = commonTexts.common.age_groups;
   const formatCoveragePercentage = useVaccineCoveragePercentageFormatter();
 
   const secondaryContent = mapData
     .sort((a, b) => {
       const age1 = Number(a.age_group_range.replace(/\D/g, ''));
       const age2 = Number(b.age_group_range.replace(/\D/g, ''));
-      return age1 - age2;
+      return age2 - age1;
```

Comment:
Could you add a comment that this logic only coincidentally matches the business logic?
Something like this

```suggestion
      return age2 - age1; // This logic only coincidentally matches the business logic because for both campaigns the older age group happens to be the primary focus group, which might not be the same for a new campaign
```